app/views/schools/change_sit/confirm.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: {action: :check_details}) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= @induction_tutor_form.school.name %></span>
+    <% if has_multiple_schools? %>
+      <h1 class="govuk-heading-xl">Are you sure you want to replace the induction tutor for <%= @induction_tutor_form.school.name %></h1>
+    <% else %>
+      <h1 class="govuk-heading-xl">Are you sure you want to replace the induction tutor for your school?</h1>
+    <% end %>
+    <p class="govuk-body">You can only assign one induction tutor to use this service for your school.</p>
+    <p class="govuk-body"><%= current_user.full_name %>  will no longer be able to:</p>
```

Comment:
```suggestion
    <p class="govuk-body">You will no longer be able to:</p>
```

Can this be anyone but themselves?

Reply:
No, it should only ever be them removing themselves